### DISCUSSION TITLE
add functionalities for Anaconda / Spyder IDE to make accessible python dev tool

### DISCUSSION THREAD

**Discussion by FrancoisLB** [nvaccess/nvda#14905] [General]
### Is your feature request related to a problem? Please describe.
the objective is to make "Anaconda Spyder IDE" fully accessible for python developpment.
in particular to manage :
* Spyder Editor window
* integrated Ipython console to easelly access to error message during code run 

### Describe the solution you'd like
NVDA should be fully compliante with all functionalities of Anaconda Spyder 👍 * code edition
- ipython console
- dialog box of "tool => preferences" menu
- etc...
### Describe alternatives you've considered
another would be to developp an APP module for Anaconda Spyder. but, such module does not exists on line.

### Additional context
this would be helpfull for visually impaired and blind people for python dev.

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.
This is very likely a problem with the implementation of the application, as a starting point please submit feedback to the authors to let them know about the problem.
This also seems like you are experiencing multiple bugs with how NVDA interacts with Anaconda Spyder. Instead, please fill out the bug template so we can understand more about your environment and the issue.

I am turning this issue into a discussion. 

Please report this issue to Anaconda.

Once Anaconda addresses the base issues with accessibility, please open specific, detailed remaining bugs using the bug template.

Kind Regards,
NV Access Software Developers
